tests/v2/waku_core/test_message_digest.nim

```diff
@@ -5,10 +5,8 @@ import
   stew/byteutils,
   testutils/unittests
 import
-  ../../waku/v2/waku_message,
-  ../../waku/v2/waku_message/codec,
-  ../../waku/v2/waku_message/digest,
-  ./testlib/wakucore
+  ../../../waku/v2/waku_core,
+  ../testlib/wakucore
```

Comment:
I think both 'waku_core' and 'wakucore' means the same for nim but, shall we standardize our naming convention?
I'd use `waku_core` for testlib as well.